mk/install.sh

```diff
@@ -74,7 +74,7 @@ fi
 
 ARCH=$(uname -m | sed -e 's/i.86/i386/g')
 case "${ARCH}" in
-    i386|x86_64) ;;
+    i386|x86_64|amd64) ;;
```

Comment:
FYI, this is because FreeBSD's uname returns 'amd64' for the Intel/AMD x86 architecture.